python/tvm/relay/frontend/tflite.py

```diff
@@ -2852,11 +2855,18 @@ def convert_transpose_conv(self, op):
         # weights tensor type should be UINT8 (quantization) or FLOAT32
         assert weights_tensor_type in (TensorType.INT8, TensorType.UINT8, TensorType.FLOAT32)
```

Comment:
Do you need to change these? Is bias now in FP16?

Reply:
No we don't. It is FP32. Weights are dequantized and then passed to this op.